src/main/java/com/jcabi/http/mock/MkAnswerBodyBytesMatcher.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2011-2015, jcabi.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the jcabi.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jcabi.http.mock;
+
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+/**
+ * Matcher for checking {@link MkAnswer#bodyBytes()} result.
+ * @author Peter Storch (peter.storch@gmail.com)
+ * @version $Id$
+ */
+@ToString
+@EqualsAndHashCode(callSuper = false, of = "matcher")
+final class MkAnswerBodyBytesMatcher extends TypeSafeMatcher<MkAnswer> {
+    /**
+     * The Matcher to use against the body.
+     */
+    private final transient Matcher<byte[]> matcher;
+
+    /**
+     * Ctor.
+     * @param match The matcher to use for the body
+     */
+    MkAnswerBodyBytesMatcher(final Matcher<byte[]> match) {
+        super();
```

Comment:
@pstorch OK, this check was removed in newer version of Qulice, but this project uses old one